solutions/standard/outputs.tf

```diff
@@ -0,0 +1,8 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "namespace_crn" {
+  description = "CRN representing the namespace"
+  value       = var.name != null ? module.namespace[0].namespace_crn : null
+}
```

Comment:
Suggest to also output the ID of the retention policy thats created. (https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cr_retention_policy#id)

Reply:
addressed